### PR TITLE
Fix cubeit so it does not transfer tracking info. Fixes #5533

### PR DESCRIPTION
**Pull request by scsides**
## Description
Fix cubeit so it does not transfer tracking info. Fixes #5533

## Related Issue

https://isis.astrogeology.usgs.gov/fixit/issues/5533
## Motivation and Context

This allows qview to run on images produced by the mosaic apps that have then been run through cubeit

## How Has This Been Tested?

Images with and without the tracking information/group in the labels were run through cubeit. The labels were inspected to make sure the tracking info was not transferred to the output cube. qview was run on both output with success.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by jlaura** [comment]
@scsides Can you get the PR template filled in as well.  Thanks!

File: isis/src/base/apps/cubeit/cubeit.xml [hunk 1]
```diff
@@ -79,6 +79,9 @@
        Modified Makefile of badinputs app test to truncate paths before data directory. Allows tests
        to pass when not using the default data area. Fixes #4783.
      </change>
+     <change name="Stuart Sides" date="2018-11-19">
+       Removed the tracking lable group if it exists in the output cube. Fixes #5533.
```
Comment (krlberry) [inline isis/src/base/apps/cubeit/cubeit.xml]: lable -> label

**Review by krlberry — CHANGES_REQUESTED** [review]
:+1: Looks good, except for one typo in documentation.

**Review by krlberry — APPROVED** [review]
:+1: Thanks!

**Pull request merged**